tests/lib/Integration/Resources/config/legacy.yml

```diff
@@ -4,3 +4,7 @@ parameters:
         - 'ger-DE'
 
 services:
+    # In ezpublish-kernel this is defined in bundle, however we use it in
+    # core implementation here
+    ezpublish.query_type.registry:
```

Comment:
This one needs `public: true`

Reply:
It's public by default, or?

Reply:
From Symfony 3.4+, it's private by default. Other `public: false` are to disable deprecation notices on Symfony 3.4.

Reply:
OK, but are you sure it needs that? This is for tests only, it's the same as in `ezpublish-kernel` and seems to be working fine.

Reply:
Ah, sorry, didn't see it was for tests only :octocat: